homeassistant/components/zeversolar/entity.py

```diff
@@ -0,0 +1,27 @@
+"""Base Entity for Zeversolar sensors."""
+from __future__ import annotations
+
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import ZeversolarCoordinator
+
+
+class ZeversolarEntity(
+    CoordinatorEntity[ZeversolarCoordinator],
+):
+    """Defines a base Zeversolar entity."""
```

Comment:
For new integrations we require them to adhere to a modern naming style.
This integration seems to comply, but is missing the marker for it.

```suggestion
    """Defines a base Zeversolar entity."""
    
    _attr_has_entity_name = True
```